src/pickers/BenefitPlanPicker.js

```diff
@@ -50,9 +50,15 @@ function BenefitPlanPicker(props) {
     { skip: true },
   );
 
-  const benefitPlans = data?.benefitPlan?.edges
-    .map((edge) => edge.node)
-    .filter((node) => node.type === type) ?? [];
+  let benefitPlans = [];
```

Comment:
can it be refactored to:
`let benefitPlans = [];

const edges = data?.benefitPlan?.edges?.map(edge => edge.node) ?? [];

if (type === BENEFIT_PLAN_TYPE.EVERY_TYPE) {
  benefitPlans = edges;
} else {
  benefitPlans = edges.filter(node => node.type === type);
}`